lib/patches/active_record/xml_attribute_serializer.rb

```diff
@@ -1,13 +1,18 @@
 require 'active_record/serializers/xml_serializer'
```

Comment:
@pranik I am playing with your fork locally on a new rails 5 beta2 project and I am seeing:

```
bundle exec rake db:migrate
rake aborted!
LoadError: cannot load such file -- active_record/serializers/xml_serializer
/Users/kenglish/workspace/filmfestivald/config/application.rb:7:in `<top (required)>'
/Users/kenglish/workspace/filmfestivald/Rakefile:4:in `require'
/Users/kenglish/workspace/filmfestivald/Rakefile:4:in `<top (required)>'
```
Is this supposed to working since `xml_serializer` is no longer part of rails5?